LocalMultiplayerAgent/MultiplayerServerManager.cs

```diff
@@ -50,6 +54,8 @@ public async Task CreateAndStartContainerWaitForExit(SessionHostsStartInfo start
                 }
             }
 
+            BasicAssetExtractor.Instance = new BasicAssetExtractor(_systemOperations, _logger, Globals.ZipExtension, Globals.TarExtension);
```

Comment:
>BasicAssetExtractor [](http://example.com/codeflow?start=47&length=19)

this should be initialized during startup and not here.

Reply:
updated !